Resources/Prototypes/_NF/Entities/World/Debris/asteroids.yml

```diff
@@ -0,0 +1,1078 @@
+  name: Asteroid Andesite Debris Huge
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      radius: 28
+      floorPlacements: 108
+
+- type: entity
+  id: AsteroidSalvageNFAndesiteSmall
+  parent: BaseAsteroidDebrisNFAndesite
+  name: Salvage Andesite Asteroid Small
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 15
+      floorPlacements: 100
+
+- type: entity
+  id: AsteroidSalvageNFAndesiteMedium
+  parent: BaseAsteroidDebrisNFAndesite
+  name: Salvage Andesite Asteroid Medium
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 17
+      floorPlacements: 150
+
+- type: entity
+  id: AsteroidSalvageNFAndesiteLarge
+  parent: BaseAsteroidDebrisNFAndesite
+  name: Salvage Andesite Asteroid Large
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 20
+      floorPlacements: 200
+
+- type: entity
+  id: AsteroidSalvageNFAndesiteHuge
+  parent: BaseAsteroidDebrisNFAndesite
+  name: Salvage Andesite Asteroid Huge
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 23
+      floorPlacements: 250
+
+# Basalt
+
+- type: entity
+  id: BaseAsteroidDebrisNFBasalt
+  parent: BaseDebrisNF
+  name: Salvage Basalt Asteroid
+  abstract: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      floorTileset:
+      - FloorBasalt
+      blobDrawProb: 0.5
+      radius: 6
+      floorPlacements: 16
+    - type: SimpleFloorPlanPopulator
+      entries:
+        FloorBasalt:
+          - id: RandomArtifactSpawner20
+            prob: 0.01
+            orGroup: rock
+          - id: BasaltElementalSpawner
+            prob: 0.08
+            orGroup: rock
+          - id: WallRockBasalt
+            prob: 0.65
+            orGroup: rock
+          - id: WallRockBasaltPlasma
+            prob: 0.15
+            orGroup: rock
+          - id: WallRockBasaltUranium
+            prob: 0.1
+            orGroup: rock
+          - id: WallRockBasaltSilver
+            prob: 0.05
+            orGroup: rock
+          - id: WallRockBasaltTin
+            prob: 0.05
+            orGroup: rock
+          - id: WallRockBasaltBananium
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockBasaltGold
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockBasaltQuartz
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockBasaltSalt
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockBasaltCoal
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockBasaltArtifactFragment
+            prob: 0.01
+            orGroup: rock
+          - id: WallBasaltCobblebrick
+            prob: 0.1
+            orGroup: rock
```

Comment:
```suggestion
            orGroup: rock
    - type: IFF
      flags: HideLabel
      color: "#b5b5b5"
```

Basalt rocks currently inherit their map colour from the base.  This suggested colour is a grey, to match the grey of the rock.  How this looks in-game:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/49aa4b1b-62a3-4172-8ffc-940880f23f2d)
The rock is readable against the UI rings and the nearby Pioneer's outline.